lib/usecases/tag/GetLogsByTagUseCase.js

```diff
@@ -33,15 +30,11 @@ class GetLogsByTagUseCase {
      * @returns {Promise} Promise object represents the result of this use case.
      */
     async execute(dto) {
-        const queryBuilder = new QueryBuilder();
         const { params } = dto;
         const { tagId } = params;
 
-        queryBuilder.where('id').is(tagId);
-        return TransactionHelper.provide(async () => {
-            const tag = await TagRepository.findOne(queryBuilder);
-            return tag ? LogRepository.findAllByTagId(tag.id) : null;
```

Comment:
@MauritsioRK , The findOne step is required to determine if a log actually exists.